lib/blueberry_rails/app_builder.rb

```diff
@@ -158,7 +157,7 @@ def install_devise
     end
 
     def setup_capistrano
-      template 'Capfile', 'Capfile'
+      copy_file 'Capfile', 'Capfile'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.